### DISCUSSION TITLE
can swiper make process dead? (with nextjs)

### DISCUSSION THREAD

**Discussion by prunusnira** [nolimits4web/swiper#7153] [Q&A]
i found that swiper with data empty or null can cause whole process become dead while rendering...
it keeps waiting for data maybe... so port is recognized as 'already in use' state.
also pm2 does not recognize it as dead (status online)

is this purposed or bug?
we are currently working on processing swiper to be hidden when data is empty or null

**Comment by hajirasyafi** [comment]
Please provide some sample code or screenshot. FYI Nextjs using client-side and server-side rendering... consider it.
